packages/gasket-plugin-redux/package.json

```diff
@@ -4,6 +4,11 @@
   "description": "Gasket Redux Setup",
   "main": "lib",
   "types": "lib/index.d.ts",
+  "files": [
+    "generator",
+    "lib",
+    "!*.spec.js"
```

Comment:
we should probably create a backlog item to standardize to /test as the other packages are